frontend/package.json

```diff
@@ -4,40 +4,30 @@
     "private": true,
     "dependencies": {
         "@babel/runtime": "7.4.2",
-        "@cthit/react-digit-components": "^1.0.0-beta.13",
-        "@date-io/date-fns": "^1.1.0",
-        "@material-ui/core": "^3.9.2",
-        "@material-ui/icons": "^3.0.2",
+        "@cthit/react-digit-components": "^1.0.0-beta.41",
+        "@material-ui/core": "^4.4.2",
+        "@material-ui/icons": "^4.2.1",
         "axios": "^0.18.0",
-        "date-fns": "2.0.0-alpha.27",
-        "formik": "^1.5.1",
-        "jss": "^9.8.7",
-        "lodash": "^4.17.11",
         "material-ui-image": "^3.1.0",
-        "material-ui-pickers": "^2.2.4",
         "netlify-cli": "^2.11.23",
-        "prop-types": "^15.7.2",
         "react": "^16.8.4",
         "react-dom": "^16.8.4",
         "react-jss": "^8.6.1",
-        "react-markdown": "^4.0.6",
-        "react-redux": "^6.0.1",
+        "react-redux": "^7.1.0",
         "react-router": "^5.0.0",
         "react-router-dom": "^5.0.0",
         "react-scripts": "2.1.8",
-        "react-select": "^2.4.2",
         "redux": "^4.0.1",
         "redux-logger": "^3.0.6",
         "redux-mock-store": "^1.5.3",
-        "redux-thunk": "^2.3.0",
         "serve": "^11.0.2",
         "styled-components": "^4.1.3",
-        "universal-cookie": "^4.0.2",
-        "yup": "^0.27.0"
+        "universal-cookie": "^4.0.2"
```

Comment:
What is this used for?

Reply:
Creates, reads, edits and deletes cookies. Saving the token to authenticate with the backend.